src/go/rdctl/cmd/extensionLS.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright © 2023 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cmd implements the rdctl commands
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// lsCmd represents the ls command
+var lsCmd = &cobra.Command{
+	Use:   "ls",
+	Short: "List currently installed images",
+	Long:  `List currently installed images.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) > 0 {
+			return fmt.Errorf("rdctl extension ls takes no additional arguments, got %s", args)
+		}
```

Comment:
```suggestion
    Args: cobra.NoArgs,
	RunE: func(cmd *cobra.Command, args []string) error {
```

Reply:
This is weird, I expected `cobra.NoArgs` to be a shortcut for `cobra.ExactArgs(0)`, which does show a reasonable error:

```console
$ rdctl extension ls x
Error: accepts 0 arg(s), received 1
Usage:
  rdctl extension ls [flags]
...
```
But we can leave this as-is.